test/iterator/ostream_iterator.cpp

```diff
@@ -49,7 +49,7 @@ int main() {
 	static_assert(models::Same<I&, decltype(*i)>);
 	static_assert(models::Same<I&, decltype(*i = 42)>);
 	static_assert(models::Same<I&, decltype(++i)>);
-	static_assert(models::Same<I, decltype(i++)>);
+	static_assert(models::Same<I&, decltype(i++)>);
```

Comment:
I'm guessing N4620 is the next revision of the Ranges working draft?  I don't see it published yet.  I see what you are referring to in N4569 though.